pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>22</version>
     </parent>
 
-    <groupId>io.takari.maven.plugins</groupId>
+    <groupId>com.facebook.presto</groupId>
```

Comment:
@electrum wanted to have this change. Its not related to this PR. I can create a new PR if required.